### PR TITLE
Update packages and eclasses from 2017

### PR DESCRIPTION
**Pull request by krnowak**
From user visible changes - it updates - whois, rpcbind, socat, dosfstools, thin-provisioning-tools, lsscsi and man-db.

~Blocked on #300.~

Needs to be merged together with https://github.com/flatcar-linux/coreos-overlay/pull/1681.

CI: http://localhost:9091/job/os/job/manifest/5080/cldsv/

### CODE REVIEW

**Comment by krnowak** [comment]
CI passed.

File: changelog/updates/2022-03-01-update-pkgs-2017.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- whois ([5.5.11](https://github.com/rfc1036/whois/commit/5f5ba8312c04a759dad05723c035549273d07461))
+- rpcbind ([1.2.6](https://git.linux-nfs.org/?p=steved/rpcbind.git;a=shortlog;h=refs/tags/rpcbind-1_2_6))
+- socat ([1.7.4.3](https://repo.or.cz/socat.git/blob/refs/tags/tag-1.7.4.3:/CHANGES))
+- dosfstools ([4.2](https://github.com/dosfstools/dosfstools/releases/tag/v4.2))
+- thin-provisioning-tools ([0.9.0](https://github.com/jthornber/thin-provisioning-tools/blob/d6d93c3157631b242a13a81d30f75453e576c55a/CHANGES#L1-L9))
+- lsscsi ([0.32](https://sg.danny.cz/scsi/lsscsi.ChangeLog))
+- man-db ([2.9.4](https://gitlab.com/cjwatson/man-db/-/tags/2.9.4))
```
Comment (tormath1) [inline changelog/updates/2022-03-01-update-pkgs-2017.md:7]: I'm curious about the `man-db` thing. `man` is not shipped in the OS but in the SDK. Should we skip this entry then ?

**Review by tormath1 — APPROVED** [review]
It looks good to me - I mainly checked for new useflags, size of the final image and CI result. Thanks a lot !

File: changelog/updates/2022-03-01-update-pkgs-2017.md (hunk 1 above)
Comment (krnowak) [reply changelog/updates/2022-03-01-update-pkgs-2017.md:7]: Hm, usually what I do to find out whether a package ends up in OS is to see the output of the build_packages job. So I search for `sys-apps/man-db` in the [packages matrix job output](http://localhost:9091/job/os/job/board/job/packages-matrix/8277/consoleText). That's why I have put that in the changelog.

Comment (tormath1) [reply changelog/updates/2022-03-01-update-pkgs-2017.md:7]: I personally look in the following file: `flatcar_production_image_packages.txt`

and `man-db` is not in this list. If we have a look to the binaries provided by this package too:
```
equery f man-db | grep bin
/usr/bin
/usr/bin/apropos
/usr/bin/catman
/usr/bin/lexgrog
/usr/bin/man
/usr/bin/man-recode
/usr/bin/mandb
/usr/bin/manpath
/usr/bin/whatis
/usr/libexec/man-db/globbing
/usr/sbin
/usr/sbin/accessdb
``` 

no one is in the final image.

Comment (dongsupark) [reply changelog/updates/2022-03-01-update-pkgs-2017.md:7]: (sorry for being late here)
+1 for what @tormath1 said here.
If specific binaries are not shipped in the production images like man-db, then we should not get that included in the changelog.

Comment (krnowak) [reply changelog/updates/2022-03-01-update-pkgs-2017.md:7]: Yeah, next time I'll use a different source of truth.

**Pull request merged**